ddtrace/tracer.py

```diff
@@ -127,16 +127,22 @@ def configure(self, enabled=None, hostname=None, port=None, uds_path=None, dogst
         if settings is not None:
             filters = settings.get(FILTERS_KEY)
 
-        if sampler is not None:
-            self.sampler = sampler
-
         # If priority sampling is not set or is True and no priority sampler is set yet
         if priority_sampling in (None, True) and not self.priority_sampler:
             self.priority_sampler = RateByServiceSampler()
         # Explicitly disable priority sampling
         elif priority_sampling is False:
             self.priority_sampler = None
 
+        if sampler is not None:
+            self.sampler = sampler
+            # TODO: Remove when we remove the fallback to priority sampling
+            if isinstance(self.sampler, DatadogSampler):
+                if self.priority_sampler:
```

Comment:
I imagine that if this is `False` then it's because `self.priority_sampler` is `None`?
So isn't `self.sampler._priority_sampler = self.priority_sampler` working in all cases?

Reply:
... fair enough!

Reply:
I also realized, this should be probably be dedented outside of this `if sampler is not None` statement, since we want to make sure if we configure a priority sampler but not a sampler, that we also update the configured `DatadogSampler`